pinot-core/src/main/java/org/apache/pinot/core/query/reduce/BrokerReduceService.java

```diff
@@ -182,7 +187,9 @@ public BrokerResponseNative reduceOnDataTable(@Nonnull BrokerRequest brokerReque
     brokerResponseNative.setNumGroupsLimitReached(numGroupsLimitReached);
     if (numConsumingSegmentsQueried > 0) {
       if (minConsumingFreshnessTimeMs == Long.MAX_VALUE) {
-        LOGGER.error("Invalid lastIndexedTimestamp across {} consuming segments", numConsumingSegmentsQueried);
+        LOGGER.error("Invalid freshness time across {} consuming segments", numConsumingSegmentsQueried);
```

Comment:
If numConsumingSegmentsQueried > 0, minConsumingFreshnessTimeMs will never be invalid.